CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+4.52.0
+==========
+### New Features
+- The improved experience using Windows broker (WAM) is now out of preview and is enabled by default on all desktop platforms, except UWP. See [3375](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3375), [3447](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3447).  
```

Comment:
@localden I will also add the link to the dev blog post here once that's published.